changelog/update-9564-transaction-timeline-deposit-to-payout-rename

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
```

Comment:
```suggestion
Type: dev
```

Reply:
Keeping unchanged for now [ref](https://github.com/Automattic/woocommerce-payments/pull/9606#discussion_r1834133295).